pkgs/development/python-modules/django-anymail/default.nix

```diff
@@ -0,0 +1,37 @@
+{ 
+    stdenv,
+    buildPythonPackage,
+    fetchPypi,
+    six,
+    requests,
+    django,
+    boto3,
+}:
+
+buildPythonPackage rec {
+  pname = "django-anymail";
+  version = "6.1.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1a3hqchi1kl12522fz2dxvym1j9n33d9ahlwzxqnf0q9w1frqz4l";
+  };
+
+  propagatedBuildInputs = [
+    six
+    requests
+    django
+    boto3
+  ];
+
+  doCheck = false;
```

Comment:
```
  src = fetchFromGitHub {
    owner = "anymail";
    repo = pname;
    rev = "v6.1";
    sha256 = "04jgz3qnsnba18rlqgxyb2g9128pk3ivflnj6695kibxg724fcpv";
  };
```

```
  checkInputs = [ mock ];
  checkPhase = ''
    substituteInPlace setup.py --replace "tests_require=[" "tests_require=[], #"
    export CONTINUOUS_INTEGRATION=1
    export ANYMAIL_SKIP_TESTS="sparkpost"
    ${python.interpreter} setup.py test
  '';
```

Reply:
dude, you're a magician. thanks for spending time on this and showing me how to do it in the future!

Reply:
in general, if I'm patching patching tests, wouldn't the `substituteInPlace` delete important requirement that break the tests?

Reply:
A trick you can often follow if you're having trouble with tests is to look at what their CI does. `.travis.yml` and `appveyor.yml` are a goldmine.